extensions/tabs/matrix.css

```diff
@@ -0,0 +1,85 @@
+@keyframes buttonTransition {
+	from {
```

Comment:
Might need to add *.css to .editorconfig so we stick with the space-instead-of-tabs that's currently be applied throughout the codebase.